iranian_cities/models.py

```diff
@@ -110,19 +85,13 @@ class RuralDistrict(models.Model):
         related_name='ruraldistricts',
```

Comment:
Fix this related name also

Reply:
@sepehr-akbarzadeh, thank you for your valuable feedback! I've implemented the changes based on your suggestions. Please review the updated code and let me know if there are any further adjustments needed.